Scripts/datatypes/person.py

```diff
@@ -119,3 +120,31 @@ def add_tours(self, purposes, tour_probs):
                 if random.random() < non_home_prob:
                     non_home_tour = Tour(purposes["oo"], tour)
                     self.tours.append(non_home_tour)
+
+    def write_file(self, resultdata, filename):
```

Comment:
I find the passing of `resultdata` and `filename` to this function a bit unnecessary. Instead we could return the line as a `str` and run `resultdata.print_line()` in `ModelSystem`. 

And a very pythonic name for the function would then be `__str__()`.